nav2_bt_navigator/behavior_trees/navigate_w_recovery_and_replanning_only_if_path_becomes_invalid.xml

```diff
@@ -0,0 +1,44 @@
+
+<!--
+  This Behavior Tree replans the global path only if the path becomes invalid and it also has
+  recovery actions specific to planning / control as well as general system issues.
+  This will be continuous if a kinematically valid planner is selected.
+-->
+<root main_tree_to_execute="MainTree">
```

Comment:
I think you need to update your documentation PR (I already merged it, so open another with just the text change) for the new XML file here with the throttling & description of why there is throttling (so we only check on a reasonable rate in case there are cascading issues we don't want to have it replan at 100hz)

Reply:
This was already correct in the documentation. The file that got updated is navigate_w_replanning_only_if_path_becomes_invalid.xml which is not in the documentation because it is not one of the main BTs.